packages/playwright-core/src/server/chromium/chromium.ts

```diff
@@ -217,7 +218,19 @@ export class Chromium extends BrowserType {
       const capabilities = value.capabilities;
       let endpointURL: URL;
 
-      if (capabilities['se:cdp']) {
+      if (process.env.SELENIUM_WS_ENDPOINT) {
```

Comment:
You should instead fix selenoid to return the endpoint in the se:cdp capability!

Reply:
You agree that this capability is not described in any specification and is a specific only for selenium (this is indicated by the `se` prefix)? This means that all services that provide access to browsers must support this capability.

I can support it for selenoid if you don't want support additional environment variable.